Examples/Example/Example/App.swift

```diff
@@ -0,0 +1,128 @@
+//
+//  AppState.swift
+//  Example
+//
+//  Created by Michael Kao on 31.10.20.
+//
+
```

Comment:
Most of this package files don't have the header. Remove it?
Same in SceneDelegate.swift